docs/network-flow-visibility.md

```diff
@@ -23,29 +25,29 @@
         - [Service Customization](#service-customization-1)
         - [Performance Configuration](#performance-configuration)
       - [Persistent Volumes](#persistent-volumes)
-  - [Pre-built Dashboards](#pre-built-dashboards)
+  - [Pre-built Grafana Dashboards](#pre-built-grafana-dashboards)
     - [Flow Records Dashboard](#flow-records-dashboard)
     - [Pod-to-Pod Flows Dashboard](#pod-to-pod-flows-dashboard)
     - [Pod-to-External Flows Dashboard](#pod-to-external-flows-dashboard)
     - [Pod-to-Service Flows Dashboard](#pod-to-service-flows-dashboard)
     - [Node-to-Node Flows Dashboard](#node-to-node-flows-dashboard)
     - [Network-Policy Flows Dashboard](#network-policy-flows-dashboard)
-  - [Dashboards Customization](#dashboards-customization)
+  - [Dashboard Customization](#dashboard-customization)
 <!-- /toc -->
 
 ## Overview
 
-[Antrea](https://github.com/antrea-io/antrea/blob/main/docs/design/architecture.md)
-is a Kubernetes network plugin that provides network connectivity and security
-features for Pod workloads. Considering the scale and dynamism of Kubernetes
-workloads in a cluster, Network Flow Visibility helps in the management and
-configuration of Kubernetes resources such as Network Policy, Services, Pods
-etc., and thereby provides opportunities to enhance the performance and security
-aspects of Pod workloads.
+Theia is a network observability and analytics platform for Kubernetes, that is
```

Comment:
Not sure if "which is" is better than "that is" or we can split it as two simple sentences.

Reply:
Sure. Let me think about how to rephrase.